### DISCUSSION TITLE
Call the stopPresenting function like the activateAR function

### DISCUSSION THREAD

**Discussion by AntonioMartinezC** [google/model-viewer#4373] [Q&A] [answered]
Hello, 

I can't find anything on calling the stopPresenting function in the same fassion as calling the activateAR function. 

The reason I want to stop the AR on click is that I want to have a button that closes AR and opens an overlay back on the "main page" without having to reload the page. I still need the Top right exit button for usability and i essentially want a second close button with extra functionality. 

Is there any way anyone has done/used a similar thing before? 

And also is there a way to create an overlay on top of the mv ar view without defining it inside the model-viewer? Giving it a real high z index didn't work and i could'nt figure out how the ar view is positioned in front of everything. Something on that would be amazing aswell.

Thank you in advance

**Comment by AntonioMartinezC** [comment]
Oh and btw I am trying to call it like so `function closeAR(button) {
    const modelViewer = button.closest('model-viewer');
    console.log("the button is clicked")
    console.log(modelViewer)
    if (modelViewer) {
    modelViewer.stopPresenting();
    }` the log logs a correct modelviewer Element from the DOM so the whole finding it works but it just says "Uncaught TypeError: modelViewer.stopPresenting is not a function"

**Comment by AntonioMartinezC** [comment, marked as answer]
Ok I have figured out a workaround but I am very open to cleaner implementations 

`function closeAR(button) {
    var modelViever = button.closest('model-viewer')
    var shadow = modelViever.shadowRoot;
    var element = shadow.querySelector('#default-exit-webxr-ar-button');
      if (element) {
        element.click();
      } else {
        console.error('Element not found: default-exit-webxr-ar-button');
      }
  }`

**Reply by elalish** [reply]
Yeah, you're outside of our public API here. Not quite sure what you're trying to do, since the normal exit AR button also doesn't reload the page. If you just want a different-looking button, you can do it with CSS here: https://modelviewer.dev/docs/index.html#entrydocs-augmentedreality-parts-defaultExitWebxrArButton

**Reply by AntonioMartinezC** [reply]
The thing I wanted to do is to have a button that would close the AR-View and then on the same click opens a modal window with more product information while still being on the same page. And yes the normal one also does not reload the page ^^

I simply wanted to have 2 buttons that can close the AR mode and have one of them do more. I think it would be nice to have that as a feature to be able to do some more UI interactions in webXR mode however I didn't quite grasp how the modelviewer exposes its functions (found an interface that has all of them but didn't get much further on that one ^^

In the end this was a fun adventure to the dark side of my DOM. If i should be more specific with what I wanted or with how I resolved it be free to ask. 

Thank you for your quick response and sorry for my late response!
